src/styles/block-switcher.scss

```diff
@@ -50,4 +50,9 @@
       box-shadow: 0 0 0 2px var(--amplify-colors-border-focus) inset;
     }
   }
+  .amplify-tabs__list {
+    overflow: hidden;
+    overflow-x: scroll;
+    scroll-padding-inline: 100%;
```

Comment:
`scroll-padding-inline` makes it so when we focus a tab (using left/right arrow keys) that is part of the overflow that it will scroll all the way into view; otherwise you could focus a tab and only see part of it. CSS IS SO NEAT THESE DAYS.

Reply:
On chrome, it doesn't seem to work for me, but I'm wondering what should it look like? 

https://github.com/aws-amplify/docs/assets/54393192/43dd4141-fefc-4b6b-a396-64d4d4314cb0

Reply:
Oh weird, mine does this after making those changes (Chrome):

https://github.com/aws-amplify/docs/assets/376920/a1e90f74-ebbf-4fda-a8f1-782118cb1206

But I did notice it's weird in Firefox so I'm going to see if there is a fix for that.